lib/phoenix_component.ex

```diff
@@ -541,6 +541,12 @@ defmodule Phoenix.Component do
         :ok
     end
 
+    {desc, opts} = Keyword.pop(opts, :desc, "")
```

Comment:
`:desc` or `:doc`?

I was also thinking we could allow it to be set to false, for example: `attr ..., doc: false`.

In this case, we can store it internally as `nil` and skip it from the docs. :)

Reply:
I like both of those suggestions!